src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Controllers/DraftApprenticeshipController.cs

```diff
@@ -104,7 +104,12 @@ private async Task AddLegalEntityAndCoursesToModel(DraftApprenticeshipViewModel
 
         private async Task<ITrainingProgramme[]> GetCourses(GetCohortResponse cohortDetails)
         {
-            var result = await _mediator.Send(new GetTrainingCoursesQueryRequest { IncludeFrameworks = !cohortDetails.IsFundedByTransfer && cohortDetails.LevyStatus != ApprenticeshipEmployerType.NonLevy });
+            var result = await _mediator.Send(new GetTrainingCoursesQueryRequest
+            {
+                IncludeFrameworks = (!cohortDetails.IsFundedByTransfer &&
+                                     cohortDetails.LevyStatus != ApprenticeshipEmployerType.NonLevy)
+                                    || cohortDetails.IsLinkedToChangeOfPartyRequest
+            });
```

Comment:
Yes, but the user can't make a selection - this is populated only so that it can be selected from the list when rendering the page to show the course name.